tests/test_detector.py

```diff
@@ -105,6 +105,32 @@ def addBadCameraSys(dw):
         self.assertRaises(lsst.pex.exceptions.InvalidParameterError,
                           DetectorWrapper, crosstalk=np.array([[1.0, 2.0], [3.0, 4.0], [5.0, 6.0]]))
 
+    def testTransform(self):
+        """Test the transform method
+        """
+        dw = DetectorWrapper()
+        pixOffset = dw.orientation.getReferencePoint()
+        for xyMM in ((25.6, -31.07), (0, 0), (-1.234e5, 3.123e4)):
+            fpPoint = afwGeom.Point2D(*xyMM)
+            pixPoint = dw.detector.transform(fpPoint, cameraGeom.FOCAL_PLANE, cameraGeom.PIXELS)
+            for i in range(2):
+                self.assertAlmostEqual(
+                    fpPoint[i]/dw.pixelSize[i] + pixOffset[i], pixPoint[i])
```

Comment:
Instead of iteration, can't you do this with `Point` and `Extent` arithmetic?

Reply:
Interesting idea. I tried `fpPoint/dw.pixelSize + pixOffset` but it was rejected with `TypeError: unsupported operand type(s) for /: 'lsst.afw.geom.coordinates.coordinates.Point2D' and 'lsst.afw.geom.coordinates.coordinates.Extent2D'`

I'll stick to this for now -- it is existing code and known to work.